### PR TITLE
Initial faucet UI

### PR DESCRIPTION
**Pull request by conqeror**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "mantis-faucet-web",
+  "version": "1.0.0",
+  "repository": "git@github.com:input-output-hk/mantis-faucet-web.git",
+  "author": "Samuel Cibulka <samuel.cibulka@gmail.com>",
+  "license": "Apache-2.0",
+  "private": false,
+  "scripts": {
+    "start": "webpack-dev-server --mode development --open",
+    "build": "rimraf dist && webpack --mode development"
```
Comment (kapke) [inline package.json]: `--mode production`?

File: src/index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Mantis Faucet</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+  </head>
+
+  <body>
+    <div class="wrapper">
+      <div id="header">
```
Comment (kapke) [inline src/index.html:11]: minor in this case - why not `header`, `main`, etc. tags?

File: src/index.html [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        <img src="images/logos-logo-horizontal-lockup.svg" id="header-logo" />
+        <span>Faucet</span>
+      </div>
+      <div id="content">
+        <form id="request-form">
+          <label for="addressField">Your address:</label>
+          <input type="text" id="address-field" name="address-field">
+          <input type="submit" value="Request funds">
+          <a id="success-message">Funds have been successfully sent to your wallet!</a>
```
Comment (kapke) [inline src/index.html]: `a`?

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  plugins: [
+    new HtmlWebpackPlugin({
+      filename: 'index.html',
+      template: 'index.html',
+    }),
+    new MiniCssExtractPlugin({
+      filename: '[name].css',
+      chunkFilename: '[id].css',
+    }),
+    new webpack.EnvironmentPlugin({
```
Comment (kapke) [inline webpack.config.js:73]: this needs to come from a cli prop or env variable to webpack as well

File: src/js/index.js [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+requestForm.onsubmit = (event) => {
+  event.preventDefault();
+  const address = document.getElementById('address-field').value
+  console.log(address)
+  client.request({method: 'faucet_sendFunds', params: [address]})
+    .then((result) => {
+      successMessage.style.display = 'block';
+    })
+    .catch((err) => {
+      errorMessage.style.display = 'block';
```
Comment (kapke) [inline src/js/index.js:20]: and log the error?

**Review by kapke — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 4 above)
Comment (conqeror) [reply webpack.config.js:73]: `EnvironmentPlugin` takes those values from `process.env`, those are just defaults.

**Review by kapke — APPROVED** [review]

**Pull request merged**